generate_protos.sh

```diff
@@ -2,7 +2,7 @@
 
 DIR=`pwd`
 rm -rf $DIR/gen
-LYFT_IMAGE="lyft/protocgenerator:5e6a3be18db77a8862365a19711428c2f66284ef"
+LYFT_IMAGE="lyft/protocgenerator:8167e11d3b3439373c2f033080a4b550078884a2"
```

Comment:
is this intentional?

Reply:
Yes. See this: https://github.com/lyft/protocgenerator/commit/8167e11d3b3439373c2f033080a4b550078884a2